hw/xbox/xbox.c

```diff
@@ -476,6 +476,28 @@ static void machine_set_eeprom(Object *obj, const char *value,
     ms->eeprom = g_strdup(value);
 }
 
+static char *machine_get_avpack(Object *obj, Error **errp)
+{
+    XboxMachineState *ms = XBOX_MACHINE(obj);
+
+    return g_strdup(ms->avpack);
+}
+
+static void machine_set_avpack(Object *obj, const char *value,
```

Comment:
We should probably support accepting the empty string here as well.

Reply:
Fixed and tested, now everything should be good to go. :ok_hand: